homeassistant/components/enocean/device.py

```diff
@@ -0,0 +1,41 @@
+"""Representation of an EnOcean device."""
+from enocean.protocol.packet import Packet
+from enocean.utils import combine_hex
+
+from homeassistant.components.enocean.const import (
+    SIGNAL_RECEIVE_MESSAGE,
+    SIGNAL_SEND_MESSAGE,
+)
+from homeassistant.helpers.entity import Entity
+
+
+class EnOceanDevice(Entity):
+    """Parent class for all devices associated with the EnOcean component."""
```

Comment:
It's a parent class for all entities not devices.

Reply:
Comment updated